.github/workflows/run-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     env:
       WSI_CONDA_CHANNEL: "https://dnap.cog.sanger.ac.uk/npg/conda/devel/generic"
       CONDA_TEST_ENVIRONMENT: "testenv"
-      PYTHON_VERSION=: "3.10"
```

Comment:
Will pick latest from `3.10` which may be newer than the one available in prod until prod is updated. But seems safe in principle.